highfidelity/hifi
# Webbrowser updates

Implementation for suggestions here:
https://highfidelity.fogbugz.com/f/cases/7001/Tablet-Web-Browser-issues-after-PR-merge

Test plan:
- Open pages with popups. Make sure popups opens
- try suggestions (only suggestions atm, no urls for suggestions)
